src/pages/SetPasswordPage.js

```diff
@@ -99,7 +99,7 @@ class SetPasswordPage extends Component {
     }
 
     render() {
-        const buttonText = !this.props.account.validated ? this.props.translate('setPasswordPage.validatingAccount') : this.props.translate('setPasswordPage.setPassword');
+        const buttonText = !this.props.account.validated ? this.props.translate('setPasswordPage.validateAccount') : this.props.translate('setPasswordPage.setPassword');
```

Comment:
It honestly has nothing to do with it, I just noticed this when i was testing and though it would be nice to update the copy since it didn't make grammatical sense.